### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by j-u-e-f**

### CODE REVIEW

**Comment by tengqm** [comment]
@j-u-e-f Please consider singing the CLA in order for this PR to be mergeable.

**Comment by Hartmarken348** [comment]
Ku,be,la,at

File: content/en/docs/concepts/workloads/pods/pod.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Force deletions can be potentially dangerous for some pods and should be perform
 
 From Kubernetes v1.1, any container in a pod can enable privileged mode, using the `privileged` flag on the `SecurityContext` of the container spec. This is useful for containers that want to use linux capabilities like manipulating the network stack and accessing devices. Processes within the container get almost the same privileges that are available to processes outside a container. With privileged mode, it should be easier to write network and volume plugins as separate pods that don't need to be compiled into the kubelet.
 
-If the master is running Kubernetes v1.1 or higher, and the nodes are running a version lower than v1.1, then new privileged pods will be accepted by api-server, but will not be launched. They will be pending state.
+If the master is running Kubernetes v1.1 or higher, and the nodes are running a version lower than v1.1, then new privileged pods will be accepted by api-server, but will not be launched. They will be in pending state.
```
Comment (shavidissa) [inline content/en/docs/concepts/workloads/pods/pod.md]: Minor suggestion: 
"If the master is running Kubernetes v1.1 or higher, and the nodes are running a version lower than v1.1, the new privileged pods are accepted by the api-server but are not be launched. They will be in the pending state."

**Pull request merged**